stats/stats-server/tests/it/mock_blockscout_reindex.rs

```diff
@@ -141,6 +146,14 @@ async fn tests_reupdate_works() {
             ("2023-03-01", "1"),
         ])
     );
+    blockscout_db.close_all_unwrap().await;
+    stats_db.close_all_unwrap().await;
+    // todo: add method for these (+timeout)
```

Comment:
hm I think I saw this method `GracefulShutdownHandler ::cancel_wait_timeout`, no?